pandas/core/internals.py

```diff
@@ -3597,7 +3597,14 @@ def value_getitem(placement):
                 return value
         else:
             if value.ndim == self.ndim - 1:
-                value = value.reshape((1,) + value.shape)
+                # gh-13012: 'reshape' is deprecated for Categorical
+                # and Series and will be removed in a subsequent version,
+                # hence the two checks done before calling 'reshape'
+                if isinstance(value, ABCSeries):
+                    value = value._values
```

Comment:
1) It was throwing warnings during testing.

2) The current code for `Series.reshape` will simply call `self._values.reshape(...)`, so I am preempting that so we don't have to the `reshape` method for `Series`.